src/Provider/IslandoraServiceProvider.php

```diff
@@ -0,0 +1,91 @@
+<?php
```

Comment:
If you look here:
https://github.com/Islandora-CLAW/Crayfish-Commons/pull/8/commits/fc681e9f1a1cd71dacaafe1d66900436253ab28c

It is a rename.